src/components/DownloadImageDialog/models.ts

```diff
@@ -0,0 +1,139 @@
+export interface Dictionary<T> {
```

Comment:
Just curious, I don't want to complicate this PR further so we can leave it as is regardless of your answer:

Did we just bring these typings over from rendition or did you go through them all and filter out anything we no longer need?

Reply:
I should have removed all unnecessary types. Next step would be to unify what is used in multiple places